tests/marqo_test.py

```diff
@@ -13,7 +13,7 @@
 
 # RUNNING CLOUD TESTS
 
-  To run Cloud V2 tests, execute `tox -e cloud_tests`.
+  To run Cloud V2 tests, execute `tox -e cloud_test_logic`.
```

Comment:
this makes more sense with the original `tox -e cloud_tests` (as we are running cloud tests, not just the cloud test logic). 
Is this possible, or are these names coupled together?

Reply:
yea it is a typo which happened during refactoring, will change